.changeset/cool-walls-drop.md

```diff
@@ -0,0 +1,5 @@
+---
+"@comet/brevo-api": minor
+---
+
+Export BrevoContactService so that it can be used in the application. This allows, for example, adding a custom REST request in the application to subscribe to the newsletter.
```

Comment:
The second sentence should be the description of the changeset

```suggestion
Export `BrevoContactsService` so that it can be used in the application

This allows, for example, adding a custom REST request in the application to subscribe to the newsletter.
```